src/Uno.UI/Generated/3.0.0.0/Windows.UI.Xaml.Controls/TextBox.cs

```diff
@@ -463,14 +463,14 @@ public  bool CanUndo
 		// Forced skipping of method Windows.UI.Xaml.Controls.TextBox.SelectionChanged.remove
 		// Forced skipping of method Windows.UI.Xaml.Controls.TextBox.ContextMenuOpening.add
 		// Forced skipping of method Windows.UI.Xaml.Controls.TextBox.ContextMenuOpening.remove
-		#if __ANDROID__ || __IOS__ || NET461 || __WASM__ || __SKIA__ || __NETSTD_REFERENCE__ || __MACOS__
+		#if false || false || false || false || false || false || false
```

Comment:
I'm not sure if it's correct to also remove `NET461` and `__NETSTD_REFERENCE__`?